src/sfizz/Region.cpp

```diff
@@ -786,6 +786,228 @@ bool sfz::Region::parseOpcode(const Opcode& rawOpcode)
         setValueFromOpcode(opcode, bendSmooth, Default::smoothCCRange);
         break;
 
+    // Modulation: LFO
+    case hash("lfo&_freq"):
+        {
+            const auto lfoNumber = opcode.parameters.front();
+            if (lfoNumber == 0)
+                return false;
+            if (!extendIfNecessary(lfos, lfoNumber, Default::numLFOs))
+                return false;
+            setValueFromOpcode(opcode, lfos[lfoNumber - 1].freq, Default::lfoFreqRange);
+        }
+        break;
+    case hash("lfo&_phase"):
+        {
+            const auto lfoNumber = opcode.parameters.front();
+            if (lfoNumber == 0)
+                return false;
+            if (!extendIfNecessary(lfos, lfoNumber, Default::numLFOs))
+                return false;
+            if (auto value = readOpcode(opcode.value, Default::lfoPhaseRange)) {
+                float normalPhase = *value * (1.0 / 360.0);
```

Comment:
Are you sure about this? The sfzformat page seems to contradict this.

Reply:
Phase is definitely expressed in degree unit.

phase=0
![LFO phase=0](https://user-images.githubusercontent.com/17614485/89971676-36a36000-dc5c-11ea-91a9-4f80a3958708.png)

phase=90
![LFO phase=90](https://user-images.githubusercontent.com/17614485/89971685-3b681400-dc5c-11ea-8a13-e258a24daa45.png)